findPeakElement.cpp

```diff
@@ -0,0 +1,34 @@
+//time complexity: O(n)
+//space complexity:O(1)
+//approach: linear search, checking if the current element is greater than left & right
+//solved on leetcode? yes, but only 54/59 test cases passed.
+//problem faced? yes, it is a brute force approach. Not all test cases passed.
+
```

Comment:
Nice try, Solve the solution taught in class. Let me know if you still find it difficult